deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobIT.java

```diff
@@ -211,7 +217,27 @@ public void ingestWorkObjectTest() throws Exception {
 
 		// Verify the properties and content of the supplemental file
 		assertBinaryProperties(supObj, FILE2_LOC, FILE2_MIMETYPE, null, FILE2_SIZE);
-
+		// Verify that ingestion event gets added for work
+		List<PremisEventObject> eventsWork = mWork.getPremisLog().getEvents();
+		assertEquals(2, eventsWork.size());
+		
+		assertTrue(eventsWork.contains(findPremisEventByEventDetail(eventsWork, "ingested as PID: " + mWork.getPid())));
```

Comment:
if you switch this to `assertNotNull` then you won't need to go through `eventsWork.contains`